src/test/java/org/springframework/cloud/deployer/spi/mesos/chronos/ChronosTaskLauncherIntegrationTests.java

```diff
@@ -37,173 +27,68 @@
-		Assert.assertThat(deploymentId1, eventually(hasStatusThat(
-				Matchers.<TaskStatus>hasProperty("state", Matchers.is(complete))), timeout.maxAttempts, timeout.pause));
-		Assert.assertThat(deploymentId2, eventually(hasStatusThat(
-				Matchers.<TaskStatus>hasProperty("state", Matchers.is(complete))), timeout.maxAttempts, timeout.pause));
-
-		((ChronosTaskLauncher)getTargetObject(taskLauncher)).cleanup(deploymentId1, deploymentId2);
+	@Override
+	protected TaskLauncher taskLauncher() {
+		return taskLauncher;
 	}
 
-	@Test
-	public void testFailedLaunch() throws Exception {
-		logger.info(String.format("Testing %s...", "FailedLaunch"));
-		Map<String, String> properties = new HashMap<>();
-		properties.put("killDelay", "1000");
-		properties.put("exitCode", "1");
-		AppDefinition definition = new AppDefinition(this.randomName(), properties);
-		Resource resource = integrationTestTask();
-		AppDeploymentRequest request = new AppDeploymentRequest(definition, resource);
-		logger.info(String.format("Launching %s...", request.getDefinition().getName()));
-		String deploymentId = taskLauncher.launch(request);
-		logger.info(String.format("Launched %s ", deploymentId));
-
-		Timeout timeout = launchTimeout();
-		Assert.assertThat(deploymentId, eventually(hasStatusThat(
-				Matchers.<TaskStatus>hasProperty("state", Matchers.is(failed))), timeout.maxAttempts, timeout.pause));
-
-		((ChronosTaskLauncher)getTargetObject(taskLauncher)).cleanup(deploymentId);
+	@After
+	public void cleanUp() {
+		for (String deploymentId : deployments) {
+			try {
+				((ChronosTaskLauncher)getTargetObject(taskLauncher)).cleanup(deploymentId);
+			}
+			catch (Exception e) {
+				log.error("Error in cleanup for {}", deploymentId, e);
+			}
+		}
 	}
 
-	/**
-	 * Tests that command line args can be passed in.
-	 */
 	@Test
-	public void testCommandLineArgs() throws Exception {
-		logger.info(String.format("Testing %s...", "CommandLineArgs"));
-		Map<String, String> properties = new HashMap<>();
-		properties.put("killDelay", "1000");
-		AppDefinition definition = new AppDefinition(this.randomName(), properties);
-		Resource resource = integrationTestTask();
-		AppDeploymentRequest request = new AppDeploymentRequest(definition, resource, Collections.emptyMap(),
-				Collections.singletonList("--exitCode=0"));
-		logger.info(String.format("Launching %s...", request.getDefinition().getName()));
-		String deploymentId = taskLauncher.launch(request);
-		logger.info(String.format("Launched %s ", deploymentId));
-
-		Timeout timeout = launchTimeout();
-		Assert.assertThat(deploymentId, eventually(hasStatusThat(
-				Matchers.<TaskStatus>hasProperty("state", Matchers.is(complete))), timeout.maxAttempts, timeout.pause));
-
-		((ChronosTaskLauncher)getTargetObject(taskLauncher)).cleanup(deploymentId);
-	}
-
-	protected String randomName() {
-		return "job-" + UUID.randomUUID().toString().substring(0, 18);
+	@Override
+	@Ignore("Currently reported as failed instead of cancelled")
```

Comment:
@trisberg Remove once https://github.com/spring-cloud/spring-cloud-deployer-mesos/issues/32 is fixed